src/Translator.php

```diff
@@ -63,7 +64,7 @@ public function exists($key)
      */
     public function plural($key, $count, array $vars = [])
     {
-        if ($count == 1) {
+        if ($count === 0 || $count === 1) {
```

Comment:
@chinpei215 Brought up a good point. `0` is plural in english, which is what this translator implements. For example you would say 'I have zero shoes' not 'I have zero shoe'.